torchao/experimental/ops/linear_8bit_act_xbit_weight/linear_8bit_act_xbit_weight.cpp

```diff
@@ -203,7 +203,8 @@ void linear_operator(
     nc = tiling_params->nc;
   } else {
     auto params = LinearTilingParams::from_target_tiles_per_thread(
-        m,
+        // We process m sequentially, so m_step is the "m" for the purpose of computing tiling params
```

Comment:
skimmed at the calculations, this would not chance params.mc but would make params.nc smaller thus improving the caching thus improving perf?